test/frontend/EmailList.test.js

```diff
@@ -0,0 +1,41 @@
+/* global describe it */
+import 'jsdom-global/register';
+import React from 'react';
+import Enzyme, { mount } from 'enzyme';
+import { expect } from 'chai';
+import EmailList from '../../src/Pages/EmailList/EmailList';
+import Adapter from 'enzyme-adapter-react-16';
+import { Button, ButtonDropdown, DropdownItem } from 'reactstrap';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('<EmailList />', () => {
```

Comment:
no test for email page or email template?

Reply:
added tests